css/style.css

```diff
@@ -0,0 +1,509 @@
+    line-height: 16px;
+  }
+
+  .content__img {
+    background-image: url(../img/small.png);
+    height: 439px;
+    width: 514px;
+    align-items: center;
+  }
+
+  .footer {
+    width: 1353px;
+  }
+}
+
+@media (max-width: 1170px) {
+
+  .wrap__site {
+    max-width: 1170px; 
+  }
+
+  .header {
+    max-width: 85%;
+    margin: 31px auto 76px;
+  }
+
+  .header__menu {
+    width: 340px;
+  }
+
+  .menu__item {
+    padding: 5px 25px 0 0;
+  }
+  
+  .footer {
+    width: 1156px;
+  }
+}
+
+@media (max-width: 940px) {
+
+  .wrap__site {
+    max-width: 940px; 
+    overflow-x: hidden;
+  }
+  
+  .header__menu,
+  .footer {
+    display: none;
+  }
+
+  .header__burger {
+    width: 35px;
+    display: flex;
+    cursor: pointer;
+  }
+
+  #header__toggle {
+    opacity: 0;
+  }
+
+  #header__toggle:checked ~ .header__btn > span {
+    transform: rotate(45deg);
+  }
+
+  #header__toggle:checked ~ .header__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+  } 
+
+  #header__toggle:checked ~ .header__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+  }
+
+  #header__toggle:checked ~ .header__menu {
+    visibility: visible;
+    right: 0;
+  }
+
+  .header__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 20px;
+    right: 30px;
+    width: 35px;
+    height: 35px;
+    z-index: 1;
+    cursor: pointer;
+  }
+
+  .header__btn > span,
+  .header__btn > span::before,
+  .header__btn > span::after {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 5px;
+    background-color: #0056EC;
+    transition-duration: .25s;
+  }
+
+  .header__btn > span::before {
+    content: '';
+    top: -9px;
+  }
+
+  .header__btn > span::after {
+    content: '';
+    top: 9px;
+  }
+
+  .header__menu {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    right: -100%;
+    width: 300px;
+    height: 100%;
+    margin: 0;
+    padding: 80px 0;
+    list-style: none;
+    transition-duration: .25s;
+  }
+
+  .menu {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .menu__item {
+    display: block;
+    padding: 12px 24px;
+    font-size: 20px;
+    font-weight: 600;
+    text-decoration: none;
+    transition-duration: .25s;
+  }
+
+  .content {
+    justify-content: center;
+    max-width: 519px;
+    height: 906px;
+    flex-flow: column-reverse;
+  }
+
+  .content__wrap {
+    height: 455px;
+    max-width: 612px;
+    margin: 0 auto;
+  }
+
+  .content__title {
+    margin: 0 auto;
+    max-width: 400px;
+    height: 170px;
+    font-size: 50px;
+    line-height: 55px;
+    text-align: center;
+    margin-bottom: 65px;
+  }
+
+  .content__desc {
+    margin: 0 auto;
+    max-width: 275px;
+    height: 57px;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    margin-bottom: 40px;
+  } 
+
+  .content__bt {
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 548px) {
+
+  .wrap__site {
+    max-width: 548px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)